apple/internal/macos_rules.bzl

```diff
@@ -216,7 +246,7 @@ def _macos_application_impl(ctx):
             platform_prerequisites = platform_prerequisites,
             provisioning_profile = provisioning_profile,
             rule_descriptor = rule_descriptor,
-            targets = ctx.attr.deps + embedded_targets,
+            targets = ctx.attr.deps + ctx.attr.extensions + ctx.attr.frameworks,
```

Comment:
I think this should be kept the same? Otherwise `ctx.attr.xpc_services` would be left out as well.